celery_batches/__init__.py

```diff
@@ -34,11 +35,21 @@ def consume_queue(queue):
 
     """
     get = queue.get_nowait
+    requests_to_requeue = []
     while 1:
         try:
-            yield get()
+            req = get()
+            if req.eta is not None:
```

Comment:
Do we need to handle `req.countdown` as well as `req.eta` or does something, somewhere convert `countdown` into `eta`?

Seems this is done for us:

* v1 task: https://github.com/celery/celery/blob/fbda0089f08d7f2a8f00925dbc0b6e10bd779251/celery/app/amqp.py#L381-L384
* v2 task: https://github.com/celery/celery/blob/fbda0089f08d7f2a8f00925dbc0b6e10bd779251/celery/app/amqp.py#L294-L300

Reply:
(To be clear, there's nothing to do here -- I answered my own question!)

Reply:
Good to point out. I think both countdown and eta will end up as `req.eta` in the end, by the time the request is handled in celery-batches.

Reply:
Yep, I think this is fine! I was just curious if it handled one and not the other, but it seems the celery code does it already! 🎉